src/components/pages/Dashboard/Dashboard.js

```diff
@@ -17,78 +20,71 @@ const Dashboard = () => {
             accessor: 'name'
           },
           {
-            Header: 'Sex',
-            accessor: 'sex'
+            Header: 'Community Name',
+            accessor: 'communityName'
           },
           {
-            Header: 'Phone Number',
-            accessor: 'phoneNumber'
+            Header: 'Farm Location',
+            accessor: 'farmLocation'
           },
           {
-            Header: 'Acres',
-            accessor: 'acres'
+            Header: 'Phone Number',
+            accessor: 'phoneNumber'
           },
           {
-            Header: 'Crops',
-            accessor: 'crops'
+            Header: 'Guarantor Name',
+            accessor: 'guarantorName'
           },
           {
-            Header: " ",
-            accessor: 'more'
+            Header: 'Guarantor Phone Number',
+            accessor: 'guarantorPhoneNumber'
           }
         ]
       }
     ],
     []
   );
 
-  // To change with data coming from API
-  const data = React.useMemo(() => makeData(20), []);
+  const prepareData = farmers => {
+    let cleanedData = farmers.map(farmer => {
+      const farmerData = {
+        name: `${farmer.personalInfo.first_name} ${farmer.personalInfo.surname}`,
+        communityName: farmer.personalInfo.community_name,
+        farmLocation: farmer.farmInfo.location_of_farm,
+        phoneNumber: farmer.personalInfo.Phone_1,
+        guarantorName: `${farmer.guarantor.grt_first_name} ${farmer.guarantor.grt_surname}`,
+        guarantorPhoneNumber: farmer.guarantor.grt_phone
+      };
+      return farmerData;
+    });
+    return cleanedData;
+  };
+
+  React.useEffect(() => {
+    let isSubscribed = true;
+    getFarmersHandler().then(farmers => {
+      if (isSubscribed) {
+        setData(prepareData(farmers));
+      }
+    });
+    return () => (isSubscribed = false);
+  }, []);
 
   return (
     <>
-      <PageHeader leftElement={Title} rightElement={<Button text={buttonText} />} />
-      <Table columns={columns} data={data} />
+      <PageHeader
+        leftElement={Title}
+        rightElement={<Button>{buttonText}</Button>}
```

Comment:
Same thing as above. I'd comment it out until we implement the feature.

Reply:
Agreed, Ill comment it out quickly